### DISCUSSION TITLE
Is this demo page using op or pro version?

### DISCUSSION THREAD

**Discussion by mfa777** [fingerprintjs/fingerprintjs#763] [General]
Demo page: https://fingerprintjs.github.io/fingerprintjs/

From the debug information it looks like a opensource version. When I access this page with normal mode and private mode of MS edge, it shows the same `Visitor identifier` value.

But when I install my own op version and do the same test, it return different `Visitor identifier` value between normal mode and private mode. Am I missing something or it is expected result? But why demo page behave different?

By the way, I use exact same code from webpack install guidance: https://github.com/fingerprintjs/fingerprintjs/blob/master/docs/api.md#webpackrollupnpmyarn

**Comment by makma** [comment]
Hey, @mfa777!
The provided page demonstrates open-source FingerprintJS (not Pro).
I believe the different results are caused by your local localhost/http/https env. Browsers treat localhost differently, take a look at: https://github.com/fingerprintjs/fingerprintjs/issues/682#issuecomment-878934807

**Comment by mfa777** [comment]
Thanks @makma In my case I use the same URL `http://192.168.1.3:3000` on both mode, so it may not related with issue #682

But when I use Brave do the same test, I got unique `Visitor identifier`. I am not sure what happened, it seems this issue caused by two factor: My site and MS edge.

I did clear cache it doesn't work.

**Reply by Finesse** [reply]
Please share the debug information from the regular and the incognito tabs so that we can see what changes
